src/protagonist/Portal/Pages/Batches/Index.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@page "{batch}"
+@using Hydra
+@using DLCS.Model.Assets
+@model Portal.Pages.Batches.Index
+
+@{
+    var batchId = Model.Batch.Id.GetLastPathElement();
+    var percentComplete = Math.Round(Model.Batch.Count > 0 ? (double)Model.Batch.Completed / (double)Model.Batch.Count * 100.0 : 1.0);
+    var progressPct = Model.Batch.Finished.HasValue ? 100 : percentComplete;
+    ViewData["Title"] = $"Batch {batchId} - {Model.Batch.Count} images";
+}
+
+<div class="card w-100">
+    <div class="card-body">
+        @if (Model.Batch.Errors > 0)
+        {
+            <div class="row px-2">
+                <div class="alert alert-danger" role="alert">
+                    <i class="me-2" data-feather="alert-circle"></i><strong>Error!</strong>
+                    This batch contains @Model.Batch.Errors image@(Model.Batch.Errors > 1 ? "s" : null) that could not be processed
```

Comment:
I think we should stick with the old wording of `has errors` rather than `could not be processed` to avoid confusion. `could not be processed` sounds like the DLCS couldn't generate derivatives but it could be the case that we couldn't fetch the image in the first instance